e2e/txserver/zeta_tx_server.go

```diff
@@ -381,27 +381,27 @@ func (zts ZetaTxServer) DeploySystemContracts(
 }
 
 // DeployZRC20s deploys the ZRC20 contracts
-// returns the addresses of erc20 zrc20
+// returns the addresses of erc20 and spl zrc20
 func (zts ZetaTxServer) DeployZRC20s(
-	accountOperational, accountAdmin, erc20Addr string,
+	accountOperational, accountAdmin, erc20Addr string, splAddr string,
```

Comment:
Let's replace there args with the following struct

```go
type ZRCDeployment struct {
   AccountOperational string
   AcountAdmin string
   ERC20 eth.Address
   SPL20 *solana.Address // if nil - no SPL ZRC-20 is deployed
}
```

Reply:
Also, let's return `eth.Address{}` instead of a string

Reply:
I agree it should take a struct.

I think it should also return a struct to make the diff a bit smaller in the future if we need to add another return value.